Autopilot/AttitudeManager/AttitudeManager.c

```diff
@@ -911,6 +919,10 @@ void readDatalink(void){
             case SET_KIVALUES:
                 setKValues(GAIN_KI,(float*)(&cmd->data));
                 break;
+            case SET_GAINS:
+                int type = (int*) (&cmd->data);
```

Comment:
You can just push to the same branch, the pork request will update.
On Jul 2, 2016 6:59 PM, "Ian Hink" <notifications@github.com> wrote:

> In Autopilot/AttitudeManager/AttitudeManager.c
> <https://github.com/UWARG/PICpilot/pull/52#discussion_r69383575>:
>
> > @@ -911,6 +919,10 @@ void readDatalink(void){
> >              case SET_KIVALUES:
> >                  setKValues(GAIN_KI,(float*)(&cmd->data));
> >                  break;
> > +            case SET_GAINS:
> > +                int type = (int*) (&cmd->data);
>
> Sorry, when I created the merge request my push had failed and I didn't
> realize. I will push and create another merge request.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/UWARG/PICpilot/pull/52/files/6b0cf14ef2f05f65849e1b4818d3e3c134310898#r69383575>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AArikWcHTzf2wrrL6beGHwLcoxr3SVaIks5qRu06gaJpZM4JDbqG>
> .
>